CRM/Civicaseextras/Upgrader.php

```diff
@@ -0,0 +1,634 @@
+<?php
+
+/**
+ * Collection of upgrade steps.
```

Comment:
Since this file is so long I'd like to review it after my proposed changes. I'd like you to implement the suggestions in https://compucorp.atlassian.net/wiki/spaces/PCHR/pages/835092529/Add+a+CiviCRM+Upgrader

You'll need to change the Upgrader by adding the method from the gist in that document. Then you can extract a lot of the stuff in this class into separate setup steps to be run on installation.

You could also have a look at this step-by-step task for how to use the new style: https://docs.google.com/document/d/1HbjDUcCV5mgNPbOFlpRuQjMuRn-HlIbxdSccAB_Sudk/edit